subsys/bluetooth/host/direction.c

```diff
@@ -7,15 +7,113 @@
 #include <assert.h>
 
 #include <bluetooth/hci.h>
+#include <bluetooth/l2cap.h>
 #include <bluetooth/conn.h>
+#include <bluetooth/direction.h>
 #include <sys/byteorder.h>
 
+#include "hci_core.h"
 #include "conn_internal.h"
+#include "direction_internal.h"
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_CORE)
 #define LOG_MODULE_NAME bt_df
 #include "common/log.h"
 
+/* @brief Antenna information for LE Direction Finding */
+struct bt_le_df_ant_info {
+	/* Bitfiled holding optional switching and sampling rates */
+	uint8_t switch_sample_rates;
+	/* available antennae number */
+	uint8_t num_ant;
+	/* Maximum supported antennae switching pattern length */
+	uint8_t max_switch_pattern_len;
+	/* Maximum length of CTE in 8[us] units */
+	uint8_t max_cte_len;
+};
+
+static struct bt_le_df_ant_info df_ant_info;
+
+#define DF_SUPP_TEST(feat, n)                   ((feat) & BIT((n)))
+
+#define DF_AOD_TX_1US_SUPPORT(supp)             (DF_SUPP_TEST(supp, \
+						BT_HCI_LE_1US_AOD_TX))
+#define DF_AOD_RX_1US_SUPPORT(supp)             (DF_SUPP_TEST(supp, \
+						BT_HCI_LE_1US_AOD_RX))
+#define DF_AOA_RX_1US_SUPPORT(supp)             (DF_SUPP_TEST(supp, \
+						BT_HCI_LE_1US_AOA_RX))
+
+/* @brief Function sets CTE parameters for connectionless transmission
```

Comment:
You don't have to document internal functions, atleast not with doxygen.

When documenting internally it is more important to document for the developers that maintain this.
In my opinion this documentation does not provide much information since the information is already contained in the function name, the input argument name and the return type.